src/dotnet-aspnet-codegenerator/dotnet-aspnet-codegenerator.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>dotnet-aspnet-codegenerator</id>
```

Comment:
We'll need to squat on this package name since it doesn't use a reserved prefix. See https://github.com/aspnet/Coherence-Signed/wiki/New-package-Id-creation. I can help with this if you need.

Reply:
Yes please help with this. 
Thanks!

Reply:
Done.